source/connect/config-file.txt

```diff
@@ -0,0 +1,62 @@
+.. _config-file:
+
+==========================
+Compass Configuration File 
+==========================
+
+You can configure |compass| settings using a :manual:`configuration file 
+</reference/configuration-options>`. Configure settings with a configuration 
+file to help manage the command line options for your large-scale deployments. 
+
+Use Cases
+---------
+
+If you do not want your settings modified after startup, the configuration file 
+allows you to configure settings for large deployments at startup and does 
+not allow them to be modified afterwards on the |compass-short| interface. 
+
+Behavior
+--------
+
+Be aware of the following information when configuring settings with a 
+configuration file:
+
+- You must create your own configuration file. |compass-short| does not provide 
+  a default configuration file. For guidance on how to create a configuration 
+  file, see :manual:`Configure the Database </administration/configuration/#configure-the-database>`.
+
+- |compass-short| accepts EJSON or YAML configuration files.
+
+- For macOS and Linux, the configuration file is located at ``/etc/mongodb-compass.conf``.
+
+- For Windows, the configuration file is located at ``<path\to\compass>\..\mongodb-compass.cfg``.
+
+- To view configuration options, run the following command in the folder 
+  containing your |compass| executable:
+  
+  .. code-block:: sh
+
+     <path-to-Compass-executable> --help
+
+  .. note::
+
+     The name and file path of the executable depend on your operating
+     system. The preceding command is for macOS.
```

Comment:
[suggestion]

I think we can omit "The preceding command is for macOS." To my knowledge, there's nothing macOS specific in that code-block.